app/src/main/java/org/mozilla/fenix/library/history/HistoryController.kt

```diff
@@ -39,7 +40,16 @@ interface HistoryController {
      * Displays a [DeleteConfirmationDialogFragment].
      */
     fun handleDeleteTimeRange()
-    fun handleDeleteSome(items: Set<History>)
+
+    /**
+     * Hides the deleted items from the UI and triggers the UNDO snackbar, that gives the user a
+     * chance to cancel the removal. After the snackbar is gone, [DefaultHistoryController.delete]
+     * would remove the items from the data layer.
+     *
+     * @param items History and History group items to be deleted.
+     * @param groups Header to be hidden from the UI.
+     */
+    fun handleDeleteSome(items: Set<History>, groups: Set<HistoryItemTimeGroup> = setOf())
```

Comment:
I went with `handleDeleteHistoryItems`, if you don't mind. It looks like there is a pattern of `onDeleteSome` `onDeleteOne,` so I went with `onDelete`, rather than `on..Deletion`